### PR TITLE
Update AG Grid Sponsor Section

### PR DESCRIPTION
**Pull request by JamesSwinton**
I've updated the Sponsor section for AG Grid:

- Added AG Charts Logo
- Updated text
- Added styles + dark mode support

N.B - the sponsor section does not appear on small (<1525~px) devices - could we adjust this section so that it appears on all devices?

### CODE REVIEW

**Comment by alexander-akait** [comment]
@JamesSwinton Thank you, can you accept CLA?

File: src/components/Sponsors/Sponsors.jsx [hunk 1]
```diff
@@ -1,41 +1,61 @@
...
         </div>
-      </Link>
+        <div className="sponsors__link-wrapper-2">
+          <Link to="https://www.ag-grid.com/?utm_source=webpack&utm_medium=banner&utm_campaign=sponsorship">
+            <div className="sponsors__tagline">
+              Datagrid and Charting for Enterprise Applications
+              <br />
+              <span style={{ color: '#0070E0', fontSize: '0.70em' }}>
+                Learn more...
+              </span>
```
Comment (snitin315) [inline src/components/Sponsors/Sponsors.jsx:40]: Do we need this `learn more...` text as the entire section is already clickable including the logos?

Comment (JamesSwinton) [reply src/components/Sponsors/Sponsors.jsx]: Thanks, I've removed this

**Comment by JamesSwinton** [comment]
@alexander-akait We're having trouble using the EasyCLA as our assigned manager no longer works with us. I've contacted the Linux foundation to try and resolve this.

**Comment by snitin315** [comment]
> I've contacted the Linux foundation to try and resolve this.

If this is a priority PR and the resolution takes too long, I can raise a new PR to make these changes from my account to get these changes merged. Let me know

**Comment by JamesSwinton** [comment]
Thanks @snitin315 for the offer - CLA is signed now

**Review by snitin315 — APPROVED** [review]
LGTM, thanks!

> the sponsor section does not appear on small (<1525~px) devices - could we adjust this section so that it appears on all devices?

We'd gladly accept a PR for this.

**Pull request merged**